### DISCUSSION TITLE
request and response of mysterymod client

### DISCUSSION THREAD

**Discussion by Suchty** [PrismarineJS/mineflayer#1842] [Q&A]
Hello

There is a client here with which you can get advantages in the game on specific servers. 
https://docs.mysterymod.net/server_api/plugin_messaging#plugin-messaging-advanced
Here is described how the client communicates and checks if the mysterymod-client is used. Is it possible to implement this request and response?

Many thanks in advance

**Comment by Neocraftr** [comment]
I've done it https://github.com/Neocraftr/griefergames-afk-bot/blob/c7666d05101b6bab0d3f297bc83b39af99986e40/index.ts#L307
In my case `bot.client` is the mineflayer bot.

**Reply by Suchty** [reply]
Very cool. Thanks a lot :)
